torch/ao/quantization/fx/utils.py

```diff
@@ -37,6 +37,12 @@
     torch.nn.functional.instance_norm : [4],
 }
 
+CONV_FUNCTIONAL_OPS = {
```

Comment:
let's just define this twice, since quantization_patterns.py will soon be removed and after that we only need this in lowering

Reply:
I see, I had actually defined it twice before but refactored it. Ok